src/shared/destinations.ts

```diff
@@ -82,9 +82,47 @@ const destinationConfigs: DestinationConfig[] = [
     keywords: ["c"],
     generateUrl: async (packageName) => {
       const repoUrl = await getRepoUrl(packageName);
-      if (repoUrl) {
-        return `${repoUrl}/blob/master/CHANGELOG.md`;
+
+      if (!repoUrl) {
+        return undefined;
+      }
+
+      // Covers GitHub repos
+      if (repoUrl.includes("https://github.com")) {
+        const url = new URL(repoUrl);
+        url.hostname = "api.github.com";
+        url.pathname = url.pathname.replace(
+          /^\/([^/]+)\/([^/]+)/i,
+          "/repos/$1/$2/contents",
+        );
```

Comment:
Maybe something like this could read a bit better?

```js
const [, owner, repo] = repoUrl.match(/\/([^/]+)\/([^/]+)/);
const apiUrl = `https://api.github.com/repos/${owner}/${repo}/contents`
```